src/index.html

```diff
@@ -7,8 +7,78 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logo">
+      </a>
```

Comment:
```suggestion
        <img
          class="logo"
          src="./images/logo.png"
          alt="logo"
        >
      </a>
```
fix code styles everywhere
check this [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) from the previous task